tests/cross/src/test/scala/tests/pc/CompletionSuite.scala

```diff
@@ -340,6 +340,25 @@ class CompletionSuite extends BaseCompletionSuite {
     )
   )
 
+  check(
```

Comment:
Could we move it to `CompletionKeywordSuite` ?

Reply:
Ah, yes. Total oversight on my part. 👍